command/certificate/inspect.go

```diff
@@ -246,108 +287,57 @@ func inspectAction(ctx *cli.Context) error {
 				}
 				zcrts = append(zcrts, zcrt)
 			}
-			b, err := json.MarshalIndent(zcrts, "", "  ")
-			if err != nil {
-				return errors.WithStack(err)
-			}
-			os.Stdout.Write(b)
-		default:
-			return errs.InvalidFlagValue(ctx, "format", format, "text, json")
+			v = zcrts
+		}
+		b, err := json.MarshalIndent(v, "", "  ")
+		if err != nil {
+			return errors.WithStack(err)
 		}
-	} else { // Only inspect the leaf certificate.
-		var block *pem.Block
+		os.Stdout.Write(b)
+		return nil
+	default:
+		return errs.InvalidFlagValue(ctx, "format", format, "text, json")
+	}
+}
 
-		if _, addr, isURL := trimURLPrefix(crtFile); isURL {
-			peerCertificates, err := getPeerCertificates(addr, roots, insecure)
+func inspectCertificateRequest(ctx *cli.Context, block *pem.Block) error {
+	format, short := ctx.String("format"), ctx.Bool("short")
```

Comment:
we should probably InvalidFlagValue if you request format=json and short, right?

Reply:
Fixed with c95f1d3c1b56f4c1ed70c6f280d477d93bd71ba4